internal/firebase/structs/structs.go

```diff
@@ -16,3 +16,9 @@ type Registration struct {
 type NotificationCounter struct {
 	NotificationsCount int `json:"notificationsCount"`
 }
+
+//VerificationCodeMetadata DB entity for verification code metadata.
+type VerificationCodeMetadata struct {
+	VsMetadata map[string]interface{} `json:"vsMetadata"`
```

Comment:
why is this map[string]interface{}? I see that the VsMetadata are not really accessed but just passed so we don't really care what is the underlying type of that value?

Reply:
According to discussion with @vonStichen this is a collection of data which they may or may not send and the form is currently unknown (if it will ever be). So for now, `map[string]interface{}` looks like a good interpretation for me.